code/modules/religion/religion_structures.dm

```diff
@@ -43,3 +43,90 @@
 			icon = sect_to_altar.altar_icon
 		if(sect_to_altar.altar_icon_state)
 			icon_state = sect_to_altar.altar_icon_state
+
+/obj/structure/destructible/religion
+	density = TRUE
+	anchored = FALSE
+	icon = 'icons/obj/religion.dmi'
+	light_power = 2
+	var/cooldowntime = 0
+	break_sound = 'sound/effects/glassbr2.ogg'
+
+/obj/structure/destructible/religion/nature_pylon
+	name = "Orb of Nature"
+	desc = "A floating crystal that slowly heals all plantlife and holy creatures. It can be bolted in place."
+	icon_state = "nature_orb"
+	anchored = FALSE
+	light_range = 5
+	light_color = LIGHT_COLOR_GREEN
+	break_message = "<span class='warning'>The luminous green crystal shatters!</span>"
+	var/heal_delay = 20
+	var/last_heal = 0
+	var/spread_delay = 45
+	var/last_spread = 0
+
+/obj/structure/destructible/religion/nature_pylon/Initialize(mapload)
+	..()
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/destructible/religion/nature_pylon/LateInitialize()
+	. = ..()
+	START_PROCESSING(SSfastprocess, src)
+
+/obj/structure/destructible/religion/nature_pylon/Destroy()
+	STOP_PROCESSING(SSfastprocess, src)
+	return ..()
```

Comment:
Does this really need to fast process if its only acting every 2 seconds?

Reply:
Unsure personally, most of the nature pylon code was copied over from the blood-cult pylon which processes every 2.5 afaik
```
/obj/structure/destructible/cult/pylon/LateInitialize()
	. = ..()
	START_PROCESSING(SSfastprocess, src)

/obj/structure/destructible/cult/pylon/Destroy()
	STOP_PROCESSING(SSfastprocess, src)
	return ..()
```

Reply:
Removing SSfastprocess after some consideration.
If it needs re-adding in a patch it shouldnt take long